docs/source/user/settings.rst

```diff
@@ -172,6 +172,21 @@ the following to your project ``settings.py``:
         ('cisco', 'Cisco Router'),
     )
 
+``OPENWISP_RADIUS_FREERADIUS_ALLOWED_HOSTS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``[]``
+
+List of host IP addresses allowed to consume the freeradius
+API endpoints, i.e Authorize, Accounting and Postauth.
+
+The organization administrators can override this setting in
+their organization dashboard.
```

Comment:
I would change this to "This value can be overridden per organization in the organization change page" or something similar.